chunking/recursive-chunking.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Recursive Chunking
+---
+
+Recursive chunking is a method of splitting documents into smaller chunks by recursively applying a chunking strategy. This is useful when you want to process large documents in smaller, manageable pieces.
+
+```python cookbook/chunking/recursive_chunking.py
```

Comment:
```suggestion
```python
```

Reply:
We keep the file name. It's consistent throughout the docs. Also points to the cookbook in this case